collectors/scm/github/pom.xml

```diff
@@ -95,7 +95,7 @@
     <dependency>
       <groupId>com.capitalone.dashboard</groupId>
       <artifactId>core</artifactId>
-      <version>3.0.4</version>
+      <<version>3.1.2-SNAPSHOT</version>
```

Comment:
There is a extra `<` in this pom.xml which is causing the builds to fail. Can you fix it ASAP? I need to merge this and then work on moving this project to its own repository.